Update README.md (#130)
kyma-incubator/reconciler: README.md

The Reconciler is a central system to reconcile Kyma clusters.

## Run Reconciler locally (Mac OS)

Follow these steps to run Reconciler locally:

> 1. Build the reconciler

Reviewer: ```suggestion
1. Build the Reconciler image:
```

```
make build-darwin 
```

> 2. Run with default configuration (all componennts from installation/resources/components.yaml)

Reviewer: ```suggestion
2. Run Reconciler with the default configuration. By default, Reconciler installs all components listed in the `installation/resources/components.yaml` file.
```

Reviewer: Should we link to this file?

```
./bin/reconciler-darwin local 
```

> Or just run with specified components:

Reviewer: ```suggestion
You can also run Reconciler with the specified components:
```
